src/main/java/com/google/sps/servlets/DBUtilities.java

```diff
@@ -0,0 +1,53 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import com.google.sps.servlets.Property;
+
+public class DBUtilities {
+
+  public static final String INTERACTION_TABLE = "Interactions";
+
+ /**
+  * Given an HTTP request object, this method will get the correlator parameter
+  * and return it. Returns an empty string if correlator is not given.
+  */
+  public static String getCorrelator(HttpServletRequest request) {
+    String correlator = request.getParameter(Property.CORRELATOR);
+
+    if (correlator == null) {
+      return "";
+    }
+
+    return correlator;
+  }
+ 
+ /**
+  * Given the correlator and the property to update, this will update the database to reflect that the user
+  * clicked on the corresponding metric.
+  *
+  * correlator - A correlator of type String for the current user.
```

Comment:
Use javaDoc style if you are going to reference the parameters
```
/**
 * description...
 * @param correlator Description of the correlator parameter.
 */
```

You also don't need to include the type ("string") in java, since its evident from reading the code.  Thats much more helpful in js, where you can't tell from the code what the type would be

Reply:
Doesn't look like this was done